docs/core/tools/dotnet-install-script.md

```diff
@@ -90,7 +90,7 @@ The install scripts do not update the registry on Windows. They just download th
 
 - **`-Help|--help`**
```

Comment:
```suggestion
- **`--help`**
```

The `-Help` switch was for PowerShell, right?

Reply:
Good catch, thanks!